DOCKER.md

```diff
@@ -0,0 +1,34 @@
+# Viewing the AtoM docs with Docker Compose
+
+This [Access to Memory](https://www.accesstomemory.org/) documentation can be
+generated and viewed locally in a web browser with the provided Docker Compose
+configuration.  Viewing the documentation locally can be very helpful when
+making updates to quickly check for syntax or content errors.
+
+## Prerequistes
+
+**These instructions require a working installation of the Docker Engine**
+
+See [Docker Engine Install](https://docs.docker.com/engine/install/) for
+instructions on installing Docker in your operating system.
```

Comment:
Not sure if we want to link to the Docker Compose docs or if that's included in the Docker Engine Install.

Reply:
Added.